unit_testing/cli/test_cli_sct_label_vertebrae.py

```diff
@@ -12,3 +16,19 @@ def test_sct_label_vertebrae_backwards_compat(script_runner):
     logger.debug(f"{ret.stderr}")
     assert ret.success
     assert ret.stderr == ''
+
+
+def test_sct_label_vertebrae_disc():
+    subprocess.check_output("sct_label_vertebrae -i sct_testing_data/t2/t2.nii.gz "
+                            "-s sct_testing_data/t2/t2_seg-manual.nii.gz "
+                            "-initfile sct_testing_data/t2/init_label_vertebrae.txt -c t2 -ofolder label_test",
```

Comment:
My apologies, @lrouhier. I've just realized `tmpdir` has been replaced by `tmp_path`. But, my recommendations are the same.

Reply:
I don't thin it will be obsolete no ?

Reply:
Yes, sadly. `tmpdir` returns a `py.path.local` object, see: https://py.readthedocs.io/en/latest/path.html

> Note: The ‘py’ library is in “maintenance mode” and so is not recommended for new projects. Please check out pathlib or pathlib2 for path operations.

`tmp_path` returns a `pathlib.Path` object, so it will be much better supported long-term.

Reply:
Looking into this a bit further, `tmpdir` returns `py.path.local`, but from the [`py.path` module](https://py.readthedocs.io/en/latest/path.html):

> **Note:** The ‘py’ library is in “maintenance mode” and so is not recommended for new projects. Please check out pathlib or pathlib2 for path operations.

If we use `tmp_path` instead of `tmpdir`, it will return a `pathlib.Path` object, which will be much better supported in the long-term.

Reply:
done in 
4e16261